src/plasmapy/simulation/particle_tracker/particle_tracker.py

```diff
@@ -602,6 +686,17 @@ def particles_on_grid(self):
 
         return all_particles
 
+    @property
+    def _particle_kinetic_energy(self):
+        r"""
+        Return the non-relativistic kinetic energy of the particles.
+        """
+
+        # TODO: how should the relativistic case be handled?
```

Comment:
Good question... no idea how relativistic stopping works! 

How about adding an 'is relativistic' property to the base Tracker (maybe like 1% C?) and then raise an exception if you try to use stopping in that regime? Until it's explicitly handled.